subprojects/packagefiles/fmt/meson_options.txt

```diff
@@ -0,0 +1,6 @@
+option(
+  'enable-header-only',
```

Comment:
We intentionally do not use 'enable-` prefix in option names as it is an autoconfism. `header-only` says the same thing more concisely.